src/pages/Dictionary/Dictionary.jsx

```diff
@@ -10,6 +13,47 @@ const Dictionary = () => {
   const users = fetchUsers();
   console.log('entries: ', entries, '\nterms: ', terms, '\nusers: ', users);
 
+  // Create a state to hold updated likes
+  const [likes, setLikes] = useState({});
+  const [dislikes, setDislikes] = useState({});
+
+  // Function to handle like button click
+  const handleLikeState = (entry) => {
+    // Increment the like count for the entry
+    setLikes((prevLikes) => ({
+      ...prevLikes,
+      [entry.id]: (prevLikes[entry.id] || 0) + 1
```

Comment:
we dont need to store likes in object form because likes and dislikes button will be mapped for each entry

Reply:
that means we dont need an object for likes and dislikes. True and False is enough

Reply:
Also containing this giant data in one state is not good

Reply:
it causes rerendering of this whole component every time a state changes..